src/stockwatch/entities/money.py

```diff
@@ -85,3 +88,68 @@ def __rmul__(self, factor: float) -> Amount:
     def __truediv__(self, divisor: float) -> Amount:
         "Return an amount self.value_exact * (1.0/divisor)"
         return replace(self, value_exact=(self.value_exact * (1.0 / divisor)))
+
+
+@dataclass(frozen=False, order=True)
+class CurrencyExchange:
```

Comment:
this class was just moved right? No need to go through it again in detail.

Reply:
right